Clarify in documentation that legacy token exchange requires FGAP:v1 (#38694)
keycloak/keycloak: docs/guides/securing-apps/token-exchange.adoc

backwards compatible with future {project_name} versions. You can also enable both token exchange features and use them together. For example, you could use both internal-internal exchange
provided by V2 together with other use cases that are supported by V1. For more details, see this <<_standard-token-exchange-comparison,token exchange comparison>>.

> NOTE: If you still need legacy token exchange feature, you also need link:{adminguide_link}#_fine_grained_permissions[Fine-grained admin permissions version 1] enabled (FGAP:v1) because

Reviewer: ```suggestion
NOTE: If you still need the legacy token exchange feature, you also need the link:{adminguide_link}#_fine_grained_permissions[Fine-grained admin permissions version 1] enabled (FGAP:v1) because
```

> NOTE: If you still need legacy token exchange feature, you also need link:{adminguide_link}#_fine_grained_permissions_v1[Fine-grained admin permissions version 1] enabled (FGAP:v1) because

Reviewer: ```suggestion
NOTE: If you still need the legacy token exchange feature, you also need the link:{adminguide_link}#_fine_grained_permissions_v1[Fine-grained admin permissions version 1] enabled (FGAP:v1) because
```

link:{adminguide_link}#_fine_grained_permissions[version 2 (FGAP:v2)] does not have support for token exchange permissions. This is on purpose because

> token-exchange is conceptually not really an "admin" permission and hence there is not plan to add token exchange permissions to FGAP:v2.

Reviewer: ```suggestion
token-exchange is conceptually not really an "admin" permission and hence there is no plan to add token exchange permissions to FGAP:v2.
```

the required target audience | Support for single audience value. Token effectively issued to the client requested by audience parameter and using scopes of that client
s|Public clients  | Not available. Downscoping implemented by V1 can be replaced by refresh token grant| Available only to exchange token of the client itself. Effectively downscoping support only
s|Consents        | Allowed for clients with `Consent required` as long as the user is already granted consent | Not allowed for clients with *Consent required*

> s|Authorization  | Verification that the requester client must be in the audience of the `subject_token`. Integration with client policies. No Fine-grained admin permissions | Based on fine-grained admin permissions:v1

Reviewer: Using the longer version for consistency:

```suggestion
s|Authorization  | Verification that the requester client must be in the audience of the `subject_token`. Integration with client policies. No Fine-grained admin permissions | Based on fine-grained admin permissions version 1
```
